integration/examples/custom/README.md

```diff
@@ -1,5 +1,10 @@
 ### Example: use the custom builder with ko
 
+**Note:** Skaffold now includes a
+[`ko` builder](https://skaffold.dev/docs/pipeline-stages/builders/ko/).
+When you use the `ko` builder, you do not need to provide a custom build shell
+script or install the `ko` binary.
```

Comment:
We should replace this example with the buildx example.

Reply:
Agree. I was thinking to keep it for the first ko builder release, so users can compare. Then we can remove it from the next release.